specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -617,6 +756,92 @@
           "type": "string"
         }
       }
+    },
+    "PutSubscriptionRequest": {
+      "description": "The parameters required to create a new subscription.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PutSubscriptionRequestProperties",
+          "description": "Put subscription request properties."
+        }
+      }
+    },
+    "PutSubscriptionRequestProperties": {
+      "description": "Put subscription properties.",
+      "required": [
+        "displayName",
+        "skuId",
+        "billingScope"
+      ],
+      "properties": {
+        "displayName": {
+          "description": "The friendly name of the subscription.",
+          "type": "string"
+        },
+        "skuId": {
```

Comment:
What's the reason not to use the standard resource-level "sku" field for this?

Reply:
sku in this flow will be of '0001', '0002, ...

Reply:
What does "0001" or "0002" mean? How will customers know what to select? Is it possible to come up with a name format for these?

Reply:
"0001" is for _Standard Azure Plan_ and "0002" is for _DevTest Azure Plan_ - updated the description (inline with other API requests).